src/CommunicationLayer/CommDeviceControl/CommDeviceManager.h

```diff
@@ -5,20 +5,26 @@ class QUdpSocket;
 
 #include "CommDefines.h"
 #include "I_CommDevice.h"
+#include <SimpleAmqpClient/SimpleAmqpClient.h>
+#include "InternetCommDevice.h"
 
 class CommDeviceManager : public I_CommDevice
 {
     Q_OBJECT
 public:
     CommDeviceManager(QUdpSocket& udpDevice);
+    CommDeviceManager(AmqpClient::Channel::ptr_t channel, QString queueName);
```

Comment:
This doesn't make sense. The purpose of a CommDeviceManager is to manage multiple devices, but here we're forcing a user to pick to only have one? There's a longer solution to how this should have been done.

But we're not using a UDP device so we should just remove all of that stuff.